### DISCUSSION TITLE
Tags source migration for <=3.1.2 ddbbs

### DISCUSSION THREAD

**Discussion by imed-ghomari** [RafaelGB/obsidian-db-folder#724] [Q&A] [answered]
### What happened?

After the update, when I load a previously created database, I get this notification and the database doesn't load
![image](https://user-images.githubusercontent.com/38974541/210340138-47fee34e-4bf0-4d0b-8b65-fef4d3d5092e.png)

Here is the db file 
[lesson tracker.md](https://github.com/RafaelGB/obsidian-db-folder/files/10335858/lesson.tracker.md)

### What platform were you using?

Desktop

### Version of the plugin

3.1.3

### Relevant log output

```shell
plugin:dataview:19999 Uncaught (in promise) Error: Failed to parse query in 'pagePaths': Error: 
-- PARSING FAILED --------------------------------------------------

> 1 | lesson
    | ^

Expected one of the following: 

'!', '(', '-', 'csv(', 'outgoing(', file link, string, tag ('#hello/stuff')

    at DataviewApi.pagePaths (plugin:dataview:19999:19)
    at DataviewApi.pages (plugin:dataview:20021:21)
    at eval (plugin:dbfolder:1:338848)
    at Generator.next (<anonymous>)
    at eval (plugin:dbfolder:1:938)
    at new Promise (<anonymous>)
    at a (plugin:dbfolder:1:697)
    at Jm (plugin:dbfolder:1:338753)
    at eval (plugin:dbfolder:1:338472)
    at Generator.next (<anonymous>)
```

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment, marked as answer]
Hello @imeed166 ,
Check your `source_form_result`config inside the md file.

Modify source_form_result: lesson per source_form_result: "#lesson"


Since the multiTag feature this source could be corrupted for old ddbbs
